Tests/configure_and_test_integration_instances.py

```diff
@@ -1311,7 +1313,7 @@ def main():
     else:
         installed_content_packs_successfully = True
 
-    tests_for_iteration = get_tests(build.server_numeric_version, build.tests)
+    tests_for_iteration = get_tests(build)
     new_integrations, modified_integrations = get_changed_integrations(build)
```

Comment:
please merge from master - @hod-alpert modified this code

Reply:
Old news.